apps/src/templates/sectionsRefresh/sections-refresh.module.scss

```diff
@@ -2,11 +2,6 @@
 @import 'font.scss';
 @import '@cdo/apps/componentLibrary/common/styles/mixins';
 
-// In order to make typography work correctly with em units. (Design system is build for 16px default font size)
-body {
```

Comment:
This was the line causing issues in the other components in the `TeacherNavigationRouter` file.   I also don't think setting the body here is best practice.  I removed it and then adjusted other CSS files to ensure the section-setup page was accurate.

Reply:
I also looked for anywhere else where we used sections-refresh.module.scss OR anywhere where we were pulling in a component that was using this module, and double checked that all the text was right.  The good news is that this file is only connected this `SectionSetup` page AND a pop-up dialog for first time section creators.  I verified both of these.